ports/geckolib/glue.rs

```diff
@@ -356,7 +356,7 @@ pub extern "C" fn Servo_StyleSet_Drop(data: *mut RawServoStyleSet) -> () {
 }
 
 pub struct GeckoDeclarationBlock {
-    pub declarations: Option<PropertyDeclarationBlock>,
+    pub declarations: Option<Arc<PropertyDeclarationBlock>>,
```

Comment:
`Arc<_>` is needed *somewhere* to allow things like `Stylist`, `SelectorMap`, and `ApplicableDeclarations` to reference blocks without copying them.

Before this PR this was `Arc<Vec<(PropertyDeclaration, Importance)>>` (one of which was a field of `PropertyDeclarationBlock`). This PR makes it `Arc<PropertyDeclarationBlock>`, but the plan is to have it pretty soon be `Arc<DOMRefCell<PropertyDeclarationBlock>>`.

I’m not sure what you mean by refcount from the Gecko side directly, but to deal with `Arc<_>` from Gecko you need FFI back into Rust to clone or drop it (increment or decrement the refcount). (Unless you want to make assumptions about the memory representation of private implementation details of `Arc`, but I strongly recommend against that.)